adapter/config/types.go

```diff
@@ -300,7 +300,8 @@ type tracing struct {
 	ConfigProperties map[string]string
 }
 
-type metrics struct {
+// Metrics defines the configuration for metrics collection.
+type Metrics struct {
```

Comment:
why did you make it a public type?

Reply:
It needed to be passed to the InitOperator method since the metrics bind address and the metrics collector are registered in that method.